hkmc2/shared/src/test/mlscript/ucs/syntax/Do.mls

```diff
@@ -58,13 +58,6 @@ f(42)
 f(41)
 //│ = Some(41)
 
-let audits = new Set()
```

Comment:
I guess you're removing this because it's already in `Else.mls`, but that file does not have `:js` at the moment.

Reply:
Yes. I've added `:js` to that test file in 166e5d0.